packages/vats/src/mintHolder.js

```diff
@@ -10,6 +10,8 @@ import {
 /** @typedef {import('@agoric/vat-data').Baggage} Baggage */
 
 /**
+ * NOTE: "keyword" connotes initial caps constraint, which doesn't apply here.
```

Comment:
consider explaining why the imperfect name remains

Reply:
Hm. I considered it, but I don't see a concise explanation.

Changing the name wouldn't require upgrading the existing contract vats. It would just require remembering that the old ones use a different name. It might even be possible to support a new `name` property and deprecate the `keyword` property.

I suppose the real reason the `keyword` name remains is: changing it isn't necessary to address #8235.

Reply:
> require remembering that the old ones use a different name
That seems reason enough to me. 

> the keyword name remains is: changing it isn't necessary

Fair. Though with this PR that imperfect name is tech debt. We should at least document the rationale to take on the debt.